Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java

```diff
@@ -77,22 +86,25 @@ public static void cleanDatabase() throws Exception {
 
     @Test
     public void shouldAddLink() throws Exception {
-        File metaXmlFile = new File("src/test/resources/metadata/4/meta.xml");
+        int parentId = testProcessIds.get(MockDatabase.HIERARCHY_PARENT);
+        int childId = testProcessIds.get(MockDatabase.HIERARCHY_CHILD_TO_ADD);
+        File metaXmlFile = new File("src/test/resources/metadata/" + parentId + "/meta.xml");
```

Comment:
It would be good if the path or parts of the path could be generalised and reused.